opm/parser/eclipse/EclipseState/Schedule/Well/WellConnections.hpp

```diff
@@ -132,9 +153,23 @@ namespace Opm {
         void orderTRACK();
         void orderMSW();
 
+        // Exclude specific connection from WELPI CF scaling.  No action unless
+        // this connection set has been prepared for WELPI.
+        void excludeFromWellPI(const std::size_t connID);
+
         Connection::Order m_ordering = Connection::Order::TRACK;
         int headI, headJ;
         std::vector< Connection > m_connections;
+
+        // Backing data for 'WELPI'.
+        //   1. No adjustment if this set of connections has not been prepared
+        //      for WELPI (m_hasWellPIAdjustment == false, default value).
+        //
+        //   2. Otherwise, scale Connection::CF() by supplied scaling factor
+        //      for those connections that are marked in m_wellPIConnections.
+        //      Apply scaling to all connections if m_wellPIConnections.empty().
+        bool m_hasWellPIAdjustment{false};
```

Comment:
operator==, serializeObject ?

Reply:
> operator==

Added [here](https://github.com/OPM/opm-common/blob/f138e92ef37cb73bf3e32ce7d1bf42e947275137/src/opm/parser/eclipse/EclipseState/Schedule/Well/WellConnections.cpp#L624-L625)

> serializeObject

Implicitly handled by default constructor.

Reply:
I've removed these members in the latest update.